### ISSUE TITLE
Getting Training Info as values

### ISSUE THREAD

**Issue by SinclairHudson** [tflearn/tflearn#1111]
Hello,

I'm currently working on a model, and training it takes quite a lot of time. I would like to have a system so that, when the model reaches a milestone in training accuracy, that it sends me an SMS message. To do this, I need the training data as values in python, which I can then pass on to another function. IE, I would like to pass the below information to another python function:
`Training Step: 9481  | total loss: 1.37357 | time: 1.061s
| Adam | epoch: 9481 | loss: 1.37357 - acc: 0.6168 | val_loss: 1.79410 - val_acc: 0.4150 -- iter: 64/64` 
How would I go about doing this? Below is my code. If you have any suggestions for my NN architecture, please voice those as well since I'm fairly new to all this stuff. Thanks in advance!
`learning_rate = 0.0001 # slow learning rate for better accuracy
training_iters = 30000  # steps
batch_size = 64

height = 128 # 128 frequency ranges measured
print("getting the max utterance length")
length = form.get_max_word_length() # dynamic input ooooohhhh
print("max utterance length is "+str(length))
classes = 18 # ajax knows 18 words
i = 0

# Network building
print("building NN")
net = tflearn.input_data([None,length,height]) # batchsize x time x freq
net = tflearn.lstm(net, 128, dropout=0.5)
net = tflearn.fully_connected(net, classes, activation='softmax')
net = tflearn.regression(net, optimizer='adam', learning_rate=learning_rate, loss='categorical_crossentropy')
model = tflearn.DNN(net, tensorboard_verbose=3)
for x in tf.get_collection(tf.GraphKeys.TRAINABLE_VARIABLES): tf.add_to_collection(tf.GraphKeys.VARIABLES, x )
print("finished NN")
# This is the sample set. Every once and a while the model takes this test of the same 200
# words and sees how it does. This is a consitent performance metric.
test_batch_x, test_batch_y = form.get_word_batch(200, length)
while i < training_iters:
        i = i+1
        trainX, trainY = form.get_word_batch(batch_size, length)
        model.fit(trainX, trainY, n_epoch=100, validation_set=(test_batch_x, test_batch_y),
                  show_metric=True, batch_size=batch_size)
        model.save("WordModels/wordClass.model")`
